newsroom/agenda/agenda.py

```diff
@@ -386,91 +386,131 @@ def _set_event_date_range(search):
 
     should = []
 
+    # Scenario 1: Default query with only date_from
     if date_from and not date_to:
-        # Filter from a particular date onwards
-        should = gen_date_range_filter("dates.end", "gte", search.args.get("date_from"), date_from)
+        should.extend(
+            [
+                # Events that start after or on date_from
+                {
+                    "bool": {
+                        "should": [
+                            {"range": {"dates.start": {"gte": date_from}}},
```

Comment:
that part is not needed imo, we want to have events starting before or after, so no need to filter that, as long as those events end after the selected date it should be fine